src/main/java/com/shopping_app/shoppingApp/model/Request/AddressRequest.java

```diff
@@ -0,0 +1,59 @@
+package com.shopping_app.shoppingApp.model.Request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AddressRequest {
+    private String street;
+    private String city;
+    private String state;
+    private String pincode;
+    private String country;
+
+    // TODO replace getter and  setter with lombok package
```

Comment:
Can you address this TODO as well in this MR?

Reply:
done